sockeye/output_handler.py

```diff
@@ -48,11 +50,15 @@ class OutputHandler:
     Abstract output handler interface
     """
 
-    def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.inference.TranslatorOutput):
+    def handle(self,
```

Comment:
Makes sense. Its better to have these translator objects as immutable data objects. 
Yes tic should also include generation of the input object.